novawallet/Modules/WalletsList/Selection/WalletSelectionPresenter.swift

```diff
@@ -66,17 +66,25 @@ extension WalletSelectionPresenter: WalletSelectionPresenterProtocol {
             !item.isSelected else {
             return
         }
-
+        shouldShowDelegatesUpdates = false
```

Comment:
not sure for the solution

Reply:
I believe we should call presenter method when controller receives viewWillDisappear. I then presenter call interactor to clear things